ci-operator/step-registry/telco5g/ptp-cluster-setup/telco5g-ptp-cluster-setup-commands.sh

```diff
@@ -51,7 +48,27 @@ cat << EOF > ~/ocp-install.yml
   hosts: hypervisor
   gather_facts: false
   tasks:
-
+EOF
+if [[ "$JOB_TYPE" == "periodic" ]]; then
+cat << EOF >> ~/ocp-install.yml
+  - name: Check if abort file exists
+    stat:
+      path: /home/kni/abort
+    register: file_info
+    failed_when: file_info.stat.exists
+
+  - name: Handle file existence
```

Comment:
This will never execute if you put `failed_when: file_info.stat.exists` before. If file exists, it will fail in a previous task.

Reply:
fixed